README.md

```diff
@@ -16,10 +15,12 @@ for background and notes.
 
 ## Web GUI Usage
 
-First, launch the server:
+First, build plugins. See [the Makefile].  
+
+Second, go to the sim folder and launch the server:
 
 ```
-$ go run cmd/skenario/main.go
+$ go run cmd/skenario/main.go ../build/plugin-k8s ../build/plugin-k8s-vpa
```

Comment:
I suggest just giving the Makefile command.  Much easier to remember.  E.g. `make run`.